backend/spec/export_ead3_spec.rb

```diff
@@ -1579,7 +1585,7 @@ def get_xml_doc
     def get_xml_doc
       as_test_user("admin") do
         DB.open(true) do
-          doc_for_resource = get_xml("/repositories/#{$repo_id}/resource_descriptions/#{@resource.id}.xml?include_unpublished=true&include_daos=true&ead3=true", true)
+          doc_for_resource = get_xml("/repositories/#{$repo_id}/resource_descriptions/#{@resource.id}.xml?include_unpublished=true&include_daos=true&include_uri_unitids=true&ead3=true", true)
```

Comment:
same as above. We should check that the default behavior is happening both 

- if `include_uri_uinitids=true` is present 
- and if no `include_uri_unitids` param is present